infra/gcp/ensure-prod-storage.sh

```diff
@@ -57,13 +57,15 @@ function empower_group_to_fake_prod() {
 
 # The GCP project names.
 PROD_PROJECT="k8s-artifacts-prod"
-PROMOTER_TESTPROD_PROJECT="k8s-cip-test-prod"
-PROMOTER_STAGING_PROJECT="k8s-staging-cip-test"
+PROMOTER_TEST_PROD_PROJECT="k8s-cip-test-prod"
+PROMOTER_TEST_STAGING_PROJECT="k8s-staging-cip-test"
 RELEASE_TESTPROD_PROJECT="k8s-release-test-prod"
 
 ALL_PROD_PROJECTS=(
     "${PROD_PROJECT}"
-    "${PROMOTER_TESTPROD_PROJECT}"
+    "${PROD_PROJECT}-backup"
+    "${PROMOTER_TEST_PROD_PROJECT}"
+    "${PROMOTER_TEST_PROD_PROJECT}-backup"
```

Comment:
Actually, instead of `"${PROMOTER_TEST_PROD_PROJECT}-backup"` can you create 2 separate ones called

`k8s-cip-test-prod2` and `k8s-cip-test-prod2-backup` ? This is because I need a place to run the e2e test that will restore images from the latter to the former, *independent* of the existing `${PROMOTER_TEST_PROD_PROJECT}` (the existing promoter e2e tests wipe images off of that one and I don't want the two tests to collide with each other).

(If you can think of a better name, please suggest.)

Reply:
PTAL